sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -309,24 +286,17 @@ export interface SessionMessageHandlerOptions {
 }
 
 // @public
-export class SessionReceiver {
-    close(): Promise<void>;
-    getMessageIterator(): AsyncIterableIterator<ServiceBusMessage>;
+export interface SessionReceiver<ReceivedMessageT extends ReceivedMessage | ReceivedMessageWithLock> extends Receiver<ReceivedMessageT> {
+    diagnostics: {
+        peek(maxMessageCount?: number): Promise<ReceivedMessage[]>;
+        peekBySequenceNumber(fromSequenceNumber: Long, maxMessageCount?: number): Promise<ReceivedMessage[]>;
+    };
     getState(): Promise<any>;
-    get isClosed(): boolean;
-    isReceivingMessages(): boolean;
-    peek(maxMessageCount?: number): Promise<ReceivedMessageInfo[]>;
-    peekBySequenceNumber(fromSequenceNumber: Long, maxMessageCount?: number): Promise<ReceivedMessageInfo[]>;
-    receiveDeferredMessage(sequenceNumber: Long): Promise<ServiceBusMessage | undefined>;
-    receiveDeferredMessages(sequenceNumbers: Long[]): Promise<ServiceBusMessage[]>;
-    receiveMessages(maxMessageCount: number, maxWaitTimeInSeconds?: number): Promise<ServiceBusMessage[]>;
-    get receiveMode(): ReceiveMode;
-    registerMessageHandler(onMessage: OnMessage, onError: OnError, options?: SessionMessageHandlerOptions): void;
     renewSessionLock(): Promise<Date>;
-    get sessionId(): string | undefined;
-    get sessionLockedUntilUtc(): Date | undefined;
+    sessionId: string | undefined;
+    sessionLockedUntilUtc: Date | undefined;
```

Comment:
The `sessionId` and `sessionLockedUntilUtc` previously were not settable by the user. Now they are. Why?

Reply:
Good point. We should make them read-only.